### PR TITLE
Add markdown files with guidelines etc.

### PR DESCRIPTION
**Pull request by philmtd**
This adds:
* An empty changelog
* A basic contributing guide
* A description of our development setup
* Changes to the readme (images)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Contributing to O-Neko
+
+*Disclaimer: We created O-Neko as a "lab day" project and had to deal with time limits during our regular
+work hours. This led to us neglecting some best practices in order to get the project to a usable state in an adequate
+time (it still took us almost two years from the idea to now). This trade off we had to make means the project is
```
Comment (schwerlaut) [inline CONTRIBUTING.md]: "in an adequate time" sound strange to me. How about something like "in an acceptable amount of time"?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+*Disclaimer: We created O-Neko as a "lab day" project and had to deal with time limits during our regular
+work hours. This led to us neglecting some best practices in order to get the project to a usable state in an adequate
+time (it still took us almost two years from the idea to now). This trade off we had to make means the project is 
+currently in a state we would like to improve with coming contributions from ourselves and the community.*
+
+## Found a bug? Have a feature request? Just have a question?
+
+Please feel free to create issues here on GitHub if you think you found a bug or want to request a feature that is not 
+currently in O-Neko. You can also ask questions via GitHub issues.
```
Comment (schwerlaut) [inline CONTRIBUTING.md]: "a feature that O-Neko currently lacks"

File: DEV_ENVIRONMENT.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+3. Fork the O-Neko project on GitHub and clone your O-Neko git repository.
+
+4. Open the project in your IDE or editor and navigate in a terminal to the `/frontend` directory and run `npm install`.
+This will install all frontend dependencies. You should do this regularly to always have the latest dependencies installed.
+The Java dependencies are managed by Maven and should be downloaded automatically, depending on your editor.
+
+5. O-Neko relies on a MongoDB. You can run it anywhere you want, but by default the application will try to connect to a database
+running on localhost with MongoDB's default port 27017. You can easily start that MongoDB by running `docker-compose up -d` in the
+project's root directory. If you want to use another MongoDB make sure to change the settings in the `/src/main/resources/application.yaml`.
```
Comment (schwerlaut) [inline DEV_ENVIRONMENT.md]: You might want to point out how the mongo-db-access can be changed through the application.yml's settings.

File: DEV_ENVIRONMENT.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+6. Start the backend. This step may vary, depending on your IDE/editor. The main class you want to run is `io.oneko.ONekoApplication`.
+We use Spring profiles to separate useful default settings from settings we use during development. This is why you need to
+run the application with the following VM option: `-Dspring.profiles.active=development`.
+
+7. Start the frontend. This is done by running `npm start` in the `/frontend` directory. This should compile the frontend
+and open your default browser. O-Neko will be available on port 4200. This will also start a proxy server that proxies
+all requests to `/api` and `/ws` to the O-Neko backend, which runs on port 8080.
+
+8. You can now log in to O-Neko with the credentials `admin`/`admin` (if you started with a clean database) and are 
+ready to start developing.
```
Comment (schwerlaut) [inline DEV_ENVIRONMENT.md]: If think this is somehow lacking an explanation on how O-Neko accesses the kubernetes-cluster.
There are default-settings (server-ip and token) in our development-profile. Maybe we want to remove them after all.

File: DEV_ENVIRONMENT.md (hunk 3 above)
Comment (philmtd) [reply DEV_ENVIRONMENT.md]: I thought that one line in the `application.yaml` was so easy that it's not worth documenting. I could totally change that though, if you think it's necessary.

File: DEV_ENVIRONMENT.md (hunk 4 above)
Comment (philmtd) [reply DEV_ENVIRONMENT.md]: Added.

**Pull request merged**